pkg/cluster/spec/validate_test.go

```diff
@@ -346,7 +346,7 @@ func (s *metaSuiteTopo) TestCountDir2(c *C) {
 	meta := ClusterMeta{}
 	yamlFile, err := os.ReadFile(file)
 	c.Assert(err, IsNil)
-	err = yaml.UnmarshalStrict(yamlFile, &meta)
+	err = yaml.Unmarshal(yamlFile, &meta)
```

Comment:
https://github.com/apptainer/apptainer/issues/473#issuecomment-1545697498

We better keep the strict behavior(it is used in cluster edit-config), something like this would do.